src/components/exercises/SyllableTable.stories.js

```diff
@@ -6,7 +6,8 @@ import SyllableTable from './SyllableTable';
 storiesOf('exercises/SyllableTable', module).add('ne', () => (
   <SyllableTable
     sound={require('../../assets/sounds/ne.mp3')}
-    syllables={['na', 'ne', 'no', 'sa', 'se', 'so', 'ta', 'te', 'to']}
-    index={1}
+    letters={['n', 's', 't']}
+    vowels={['a', 'e', 'o']}
+    indices={{ letterIndex: 0, vowelIndex: 1 }}
```

Comment:
I would pass the two indices directly because
* then you don't need to destructure the object in `SyllableTable`
* my current plan for generating props (#91) won't cover objects. So that might be an issue if we want to test a random syllable (not sure if that is needed)